application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/MigrationSheet.xml

```diff
@@ -151,6 +151,20 @@
         #displayFormField('input', $propertyDescriptor)
       #end
     &lt;/dl&gt;
+    &lt;h2&gt;$services.localization.render('confluencepro.job.onlyrights.title')&lt;/h2&gt;
```

Comment:
missing `xml` escaping. It seems that I missed it as well in a bunch of places 😅 
```suggestion
    &lt;h2&gt;$escapetool.xml($services.localization.render('confluencepro.job.onlyrights.title'))&lt;/h2&gt;
```